furyctl-on-eks/README.md

```diff
@@ -0,0 +1,522 @@
+
+This tutorial assumes some basic familiarity with Kubernetes and AWS. Some experience with Terraform is helpful but not required.
+
+To follow this tutorial, you need:
+
+- **AWS Access Credentials** of an AWS Account with the following [IAM permissions][terraform-aws-eks-iam-permissions].
+- **Docker** - the tutorial uses a [Docker image][fury-getting-started-dockerfile] containing `furyctl` and all the necessary tools to follow it.
+- **OpenVPN Client** - [Tunnelblick][tunnelblick] (on macOS) or [OpenVPN Connect][openvpn-connect] (for other OSes) are recommended, [OpenVPN client][openvpn-client] is required
+when using the flag `--vpn-auto-connect` in the `furyctl create/delete cluster` command.
+- **AWS S3 Bucket** to store the Terraform state.
+- **GitHub** account with [SSH key configured][github-ssh-key-setup].
+- **AWS CLI** - version 2.8.12 at the time of writing this tutorial. You can check your version by running `aws --version`. If you don't have it installed, follow the [official guide][aws-cli-installation].
+
+### Setup and initialize the environment
+
+1. Open a terminal
+
+2. Clone the [fury getting started repository][fury-getting-started-repository] containing the example code used in this tutorial:
+
+```bash
+git clone https://github.com/sighupio/fury-getting-started/
+cd fury-getting-started/fury-on-eks
+```
+
+3. Run the `fury-getting-started` docker image:
+
+```bash
+docker run -ti --rm \
+  -v $PWD:/demo \
+  registry.sighup.io/delivery/fury-getting-started
+```
+
+4. Setup your AWS credentials by exporting the following environment variables:
+
+```bash
+export AWS_PROFILE=<YOUR_AWS_PROFILE_NAME>
+```
+
+If you don't have an AWS profile configured, you can create one by running the following command:
+
+```bash
+$ aws configure --profile <YOUR_AWS_PROFILE_NAME>
+AWS Access Key ID [None]: <YOUR_AWS_ACCESS_KEY_ID>
+AWS Secret Access Key [None]: <YOUR_AWS_SECRET_ACCESS_KEY>
+Default region name [None]: <YOUR_AWS_REGION>
+Default output format [None]: json
+```
+
+You are all set ✌️.
+
+## Step 1 - Configure the EKS cluster via `furyctl.yaml` 
+
+`furyctl` is a command-line tool developed by SIGHUP to support:
+
+- the automatic provisioning of Kubernetes clusters in a number of cloud environments
+- the installation of the Fury distribution
+
+The configuration of the Fury cluster is governed by the `furyctl.yaml` file, which for the purposes of this tutorial 
+is located in `/demo/furyctl.yaml`
+This file contains the information needed to set up the cluster and the
+configuration and consists of the following sections:
+
+- **infrastructure**: contains the information related to the infrastructure (VPC and VPN) provisioning phase.
+- **kubernetes**: contains the information related to the provisioning phase of the Kubernetes cluster.
+- **distribution**: contains information related to the provisioning phase of the distribution.
+
+There is also an initial part that defines settings common to all sections.
+
```

Comment:
I'd mention here that you can generate a sample configuration file to start working with the `furyctl create config <params>` command